packages/compartment-mapper/src/search.js

```diff
@@ -4,9 +4,13 @@
 
 // @ts-check
 
-/** @import {ReadFn} from './types.js' */
-/** @import {ReadPowers} from './types.js' */
-/** @import {MaybeReadPowers} from './types.js' */
+/**
+ * @import {
+ *   ReadFn,
+ *   ReadPowers,
+ *   MaybeReadPowers,
+ * } from './types.js'
+ */
```

Comment:
I notice that in all of our `@import` blocks, we do not end each statement with a semicolon.  I am unsure if this is considered inconsistent given the presence of semicolons everywhere else

Reply:
My understanding is that `@` pragmas are not generally terminated with a semicolon so I assumed they wouldn’t even be tolerated. Would have to check. My preference is to omit them given that they are not necessary.